content/authors/aquif-zubair.yml

```diff
@@ -0,0 +1,6 @@
+name: Aquif Zubair
+bio: |
+  Aquif Zubair | Fellow Jul 2020 Cohort
+avatar: ./avatars/aquif-zubair.jpg
+social:
+  - url: https://twitter.com/aquifzubair
```

Comment:
You may have to add a new line at the end of the file here @aquifzubair

Reply:
done sir. @aksanoble